### PR TITLE
Added proper configuration paramaters for specific pewpew functions

### PR DESCRIPTION
**Pull request by Mutoxicated**

### CODE REVIEW

**Comment by Tasty-Kiwi** [comment]
Remaking pr :skull:

File: generator/generate_snippets.ps1 [hunk 1]
```diff
@@ -43,10 +43,20 @@ function Out-Functions {
     $functions_obj[$func.func_name] = [ordered]@{}
     $functions_obj[$func.func_name]["prefix"] = ("{0}.{1}" -f $table_name, $func.func_name)
     for ($i = 0; $i -lt $func.parameters.Count; $i++) {
```
Comment (Tasty-Kiwi) [inline generator/generate_snippets.ps1:45]: replace this with a foreach

File: generator/generate_snippets.ps1 [hunk 2]
```diff
@@ -43,10 +43,20 @@ function Out-Functions {
...
+      $editedString = "{";
+      if ($func.parameters[$i].type -eq "Map")
+      {
+        foreach ($value in $func.paramaters[$i].map_entries) {
```
Comment (Tasty-Kiwi) [inline generator/generate_snippets.ps1]: use a better value variable, something like `$entry`

**Review by Tasty-Kiwi — CHANGES_REQUESTED** [review]

File: generator/generate_snippets.ps1 (hunk 2 above)
Comment (Tasty-Kiwi) [reply generator/generate_snippets.ps1]: nice

File: generator/generate_snippets.ps1 (hunk 1 above)
Comment (Tasty-Kiwi) [reply generator/generate_snippets.ps1:45]: nvm
i also use classical for loop here

Comment (Mutoxicated) [reply generator/generate_snippets.ps1:45]: k

**Review by Tasty-Kiwi — APPROVED** [review]

**Comment by Tasty-Kiwi** [comment]
After testing i will merge tomorrow

**Comment by Tasty-Kiwi** [comment]
So many commits :skull:

**Pull request merged**